udaykalra/portfolio/src/main/webapp/script.js

```diff
@@ -22,10 +30,10 @@ function createCommentList(inputText) {
 /**
  * Fetches comments for display.
  */
-function getCommentData() {
-  fetch('/data')                          // sends a request to /data
-      .then(response => response.json())  // parses the response as JSON
-      .then((myComments) => {  // now we can reference the fields in myObject!
+function getCommentData(commentsLimit = 20) {
+  fetch('/data?comment-count=' + commentsLimit)  // sends a request to /data
+      .then(response => response.json())         // parses the response as JSON
+      .then((myComments) => {  // now we can reference the fields as an object!
```

Comment:
Suggest renaming `myComments` to `comments`, since "my" is ambiguous here (who does it represent?).

Reply:
Done